src/App.jsx

```diff
@@ -14,33 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
+const sortFieldAlphabet = 'alphabet';
+const sortFieldLength = 'length';
+const reverseFieldGoods = 'reverse';
+
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState('');
+  const [originalGoods, setOriginalGoods] = useState(goodsFromServer);
+
+  const reset = () => {
+    setVisibleGoods(goodsFromServer);
+    setActiveSort('');
+  };
+
+  const sortByAlphabet = () => {
```

Comment:
The method name 'sortByAlphabet' does not start with a verb. According to the checklist, the method's name should start with a verb. Consider renaming it to 'sortAlphabetically'.